run-local.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+ npm run build && npm run buildsass && npm run buildjs && npm run buildgovuk && npm run buildlocalimages && node -r 'dotenv/config' dist/app.js
```

Comment:
This can be another script in `package.json` rather than a separate bash script file

Reply:
Good call.

Reply:
And done.